CONTRIBUTING.md

```diff
@@ -153,3 +158,23 @@ vscode's internal debugger.
   ]
 }
 ```
+
+## First Timers Welcome!
+
+If you are new to contributing to open source, or even new to contributing to this project specifically, we gladly welcome your contributions! Contributing directly to the codebase will require some familiarity with `node.js` and TypeScript, but if you are unfamiliar with these technologies that is okay! Here are a few other ways you can contribute to the bot!
+
+### Testing Features
+
+You can assist us with the review of Pull Request submissions and current features by testing them! The most efficient way to do so is to fork the repository, clone your fork to your local machine, and follow the steps above for setting up your Discord Bot application and configuring the code to connect to that application. However, if you are unable to run this locally (or uncomfortable/unfamiliar with the process) you can also spin up a [GitPod instance](http://gitpod.io/#https://github.com/bradtaniguchi/discord-bot-test) and test features through that as well.
+
+### Documentation
+
+Keep an eye out for open issues that are labelled with `documentation` - these are issues specifically related to the project documentation files (such as this one!). Contributing to these issues requires less knowledge of the codebase and how the code functions and more knowledge of how the bot and its features function.
+
+### Submit Feature Requests
+
+Do you have an idea for a new bot feature? _Anyone_ is welcome to submit a feature request, even if you have no idea how to write the code for that feature! Please feel free to submit your ideas as [GitHub issues]() so we may review them and determine the implementation method and use case.
+
+### Our Commitment to You
+
+We know what it is like to make your first contributions to a project. Sometimes it can feel intimidating or daunting, and that is completely normal - every developer experiences that feeling. We assure you that we will never make you feel ignorant or bad for submitting an issue or pull request. While we extensively review all pull request submissions for the code quality and functionality, we will not belittle your work. It is important, however, to keep an open mind when you submit a pull request - our review process is in place to ensure the code is consistent and high-quality, but also to help you learn and grow as a developer. If you have any questions or concerns, do not hesitate to contact us! We are more than happy to help!
```

Comment:
I'm a little unsure about committing to much here, people can get busy and we might not be able to help everyone with everything. I'm fine with stating stuff that's already listed in the code of conduct, but I don't want to promise anything. 

Might be best to remove this section. Open source is a collective effort, but not a promise (at least this project isn't ;D).

Reply:
The idea here was that we basically promise not to be jerks and laugh at you if your code isn't quality code, but review it and provide suggestions to fix it (which is what we do anyway)

Reply:
Removed :D